core/src/test/resources/test-reference.conf

```diff
@@ -163,6 +163,18 @@
       access: "commit-log"
     }
 
+    event-data-storage-with-suffix {
+      entity: event
+      attributes: [ data ]
+      # this is testing only storage
+      storage: "inmem:///proxima_events-with-suffix"
+      type: replica
+      access: "commit-log"
+      cfg: {
```

Comment:
All of this is generally a `cfg`, co I'd suggest calling this a bit differently. How about `replication`, as this configuration applies to repication controller?

Reply:
Changed